internal/declarative/v2/inmemory_rendered.go

```diff
@@ -62,6 +62,5 @@ func (c *InMemoryManifestCache) Parse(spec *Spec,
 }
 
 func generateCacheKey(spec *Spec) string {
-	file := filepath.Join(ManifestFilePrefix, spec.Path, spec.ManifestName)
-	return fmt.Sprintf("%s-%s", file, spec.Mode)
+	return filepath.Join(ManifestFilePrefix, spec.Path, spec.ManifestName) + "-raw"
```

Comment:
let's get rid of this magic `-raw` suffix, if there is no other `spec.Mode`, then it does not bring any value to contribute cache key.

Reply:
Sure, in the sense of true refactoring I didn't want to change the behavior at all. If you think it's fine to remove, will do.

Reply:
leave it like this will make it more difficult to understand in the future, no one knows what's the meaning of this `-raw`. If you want to keep the existing implementation, then it's better we also refactor it to following best practice, e.g: define a const value with a meaningful name to indicate what it means.

However, it's just a cache key, a fixed suffix not brings any difference, I mean you can't messed up while generate new cache. And all existing cache value will be anyway evicted after new release. So you don't need to care about compatible problem here.